app/views/partials/personal_key/_key.html.erb

```diff
@@ -1,21 +1,18 @@
-<div class="bg-personal-key">
-  <h2 class="center margin-top-1 padding-top-1 fs-20p">
+<div class="border-y border-primary-light bg-primary-lightest padding-y-3 center">
+  <h2 class="margin-y-0">
     <%= t('users.personal_key.header') %>
   </h2>
-  <div class="margin-top-6 margin-bottom-6 padding-x-0 tablet:padding-x-1 padding-y-2 center border-width-2px separator-text bg-pk-box">
-    <%- code.split('-').each do |word|
-      -%><div class="inline"
-        ><strong><code class="separator-text__code monospace" data-personal-key="word"
-          ><%= word
-        %></strong></code
-      ></div
-    ><%- end -%>
+  <div class="bg-personal-key padding-top-4 margin-y-2">
+    <div class="padding-x-0 tablet:padding-x-1 padding-y-2 separator-text bg-pk-box">
+      <% code.split('-').each do |word| %>
+        <strong class="separator-text__code monospace" data-personal-key="word"><%= word %></strong>
+      <% end %>
+    </div>
```

Comment:
sorry to be a party pooper but all the weird `><` split across lines was to remove spaces between words, `spec/views/shared/personal_key/_key.html.erb_spec.rb:6` is gonna get cranky:

```
partials/personal_key/_key.html.erb renders the code without whitespace between segments
     Failure/Error: expect(doc.text).to include('abcdefghijklmnop')
```

Reply:
> sorry to be a party pooper but all the weird `><` split across lines was to remove spaces between words, `spec/views/shared/personal_key/_key.html.erb_spec.rb:6` is gonna get cranky:

Hm, are we okay to just update the spec to tolerate it? (53b1a6aca)

The copy logic itself only cares about what's within the `[data-personal-key]` attribute, so assuming that combined text has no whitespace within, it should be fine?

https://github.com/18F/identity-idp/blob/b6fdcef83072140c713f5159b2ef7bd8a0a8fba5/app/javascript/packs/personal-key-page-controller.js#L7

https://github.com/18F/identity-idp/blob/b6fdcef83072140c713f5159b2ef7bd8a0a8fba5/app/javascript/packs/personal-key-page-controller.js#L15-L23

Reply:
I feel similarly, but not overwhelmingly so

Reply:
Oh, that's a good point. Let me check to see if that regressed with the changes here. I'll make it work 👍

Reply:
ooo `concat` is tricky... reminds me of PHP `<? print("aaaa") ?>` 👍